lib/Auth/Process/DatabaseCommand.php

```diff
@@ -43,41 +43,47 @@ public static function insertLogin(&$request, &$date)
         $month = $date->format('m');
         $day = $date->format('d');
 
-        if (is_null($idpEntityID) || empty($idpEntityID) || is_null($spEntityId) || empty($spEntityId)) {
+        if ($idpEntityID === null || empty($idpEntityID) || $spEntityId === null || empty($spEntityId)) {
```

Comment:
I think it is not necessary to check against NULL. According to the documentation, if the value is equal to NULL, the empty() function will return TRUE. 
Should we remove similar checks in this PR or it will be done in the next one? @pajavyskocil